redfishMockupCreate.py

```diff
@@ -668,7 +668,11 @@ def get_nav_and_collection_properties(rft, rs, exceptionList):
                         elif len(x) > 1 and '@odata.type' in x:
                             ns, ver, resType = rft.parseOdataType(rft, x)
                             if resType == 'LogEntry':
```

Comment:
Is this here just for the auto-expanded resources case? Maybe we should try to clean this up at some point to make handling of that more generic. In doing so, then we can probably have generic handling for lack of `@odata.id`.

Reply:
Yes, I think that is for the auto-expand case. I agree it could benefit from some clean-up.